velox/expression/CastExpr.cpp

```diff
@@ -103,6 +103,7 @@ VectorPtr castFromDate(
     }
     case TypeKind::TIMESTAMP: {
       static const int64_t kMillisPerDay{86'400'000};
+      const auto& queryConfig = context.execCtx()->queryCtx()->queryConfig();
```

Comment:
Why is this change needed ? Please remove it if not required.

Reply:
Not related. Removed.